classes/class-wc-connect-shipping-label.php

```diff
@@ -56,7 +56,7 @@ protected function get_packaging_data( WC_Order $order ) {
 			$shipping_methods = $order->get_shipping_methods();
 			$shipping_method = reset( $shipping_methods );
 			if ( ! $shipping_method || ! isset( $shipping_method[ 'wc_connect_packages' ] ) ) {
-				return $this->get_items_as_individual_packages( $order );
+				return array( FALSE, $this->get_items_as_individual_packages( $order ) );
```

Comment:
I don't know what I was thinking, this is really obfuscated code (maybe meant as temporal code?)

I've ended up breaking this method into 2, returning an array (of any kind) when you want a method to return 2 things doesn't feel like a good idea.